### ISSUE TITLE
Enable OOP functionality to use the 64bit host

### ISSUE THREAD

**Issue by rynowak** [aspnet/Razor#1705]
From @heejaechang

For both LUT and Razor, can you guys add some *.json files on top of your existing json files with this modification.

For each your service file (*.servicehub.service.json), add another json file with 64 ending 
ex, razorLanguageService64.servicehub.service.json for razorLanguageService.servicehub.service.json)

with this change

  "host": "desktopClr.x86" => "host": "desktopClr "
"hostId": "RoslynCodeAnalysisService32" => "hostId": "RoslynCodeAnalysisService"

Everything else should be exactly same.

And then add this new file (*64.servicehub.service.json) to all places where *.servicehub.service.json would be placed such as vsixmanifest

For Razor, that is all you need to do.

I can change this (http://source.roslyn.io/#Microsoft.VisualStudio.LanguageServices.Razor.RemoteClient/RazorLangaugeServiceClient.cs,15) to pick up right json file from A/B testing in Roslyn side for you.

For LUT, you need to do the below as well.

Basically, check some options and then choose between remoteBuildService or remoteBuildService64 
Ex) http://source.roslyn.io/#Microsoft.VisualStudio.LanguageServices/Implementation/Remote/RemoteHostClientServiceFactory.RemoteHostClientService.cs,166

**Comment by rynowak** [comment]
See: https://github.com/dotnet/roslyn/pull/21839/files

**Comment by NTaylorMullen** [comment]
https://github.com/aspnet/Razor/commit/edcf6857d1d4ad0f63f489feb2c978c8df948b31
